components/model/src/ome/model/internal/NamedValue.java

```diff
@@ -95,6 +95,28 @@ public int hashCode() {
     }
 
     @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
+        if (obj == null)
+            return false;
+        if (getClass() != obj.getClass())
+            return false;
```

Comment:
This is interesting. I was reading https://netbeans.org/bugzilla/show_bug.cgi?id=156994 and wondering if the choice here is related to `NamedValue` being mapped by Hibernate.